tracer/build/_build/NukeExtensions/CustomDotNetTasks.cs

```diff
@@ -40,4 +44,28 @@ public static IEnumerable<(DotNetMSBuildSettings Settings, IReadOnlyCollection<O
     {
         return configurator.Invoke(DotNetMSBuild, DotNetTasks.DotNetLogger, degreeOfParallelism, completeOnFailure);
     }
+
+    public static IReadOnlyCollection<Output> DotNet(string arguments, MSBuildTargetPlatform platform, string workingDirectory = null, IReadOnlyDictionary<string, string> environmentVariables = null, int? timeout = null, bool? logOutput = null, bool? logInvocation = null, bool? logTimestamp = null, string logFile = null, Func<string, string> outputFilter = null)
```

Comment:
Can we inline this to be a local function instead? AFAICT its _only_ called in `RunSampleWithSanitizer` and IMO could easily be confused with the existing `DotNet` function etc. Once we extract out `GetDotNetPath` it's really only 3 lines, so not worth having the indirection/confusion IMO